src/Installer/System/Linux/Debian/NoSudo.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Dock\Installer\System\Linux\Debian;
+
+use Dock\Installer\InstallerTask;
+use Dock\IO\ProcessRunner;
+use Dock\IO\UserInteraction;
+use SRIO\ChainOfResponsibility\DependentChainProcessInterface;
+
+class NoSudo extends InstallerTask implements DependentChainProcessInterface
+{
+    /**
+     * @var ProcessRunner
+     */
+    private $processRunner;
+    /**
+     * @var UserInteraction
+     */
+    private $userInteraction;
+
+    /**
+     * @param UserInteraction $userInteraction
+     * @param \Dock\IO\ProcessRunner $processRunner
+     */
+    public function __construct(UserInteraction $userInteraction, ProcessRunner $processRunner)
+    {
+        $this->userInteraction = $userInteraction;
+        $this->processRunner = $processRunner;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function dependsOn()
+    {
+        return ['docker'];
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getName()
+    {
+        return 'noSudo';
+    }
+
+    public function run()
+    {
+        if (! $this->processRunner->run('groups | grep docker', false)->isSuccessful()) {
```

Comment:
Can you make it more explicit by moving this logic into a method name, for instance, `isCurrentUserInDockerGroup`.